File: FizzBuzz/FizzBuzz.js
// FizzBuzz Kata 
/*     
if i is a multiple of 3 print "Fizz"
else if i is a multiple of 5 print "Buzz"
else if i is a multiple of 3 and a multiple of 5 print "FizzBuzz"
else print i
*/



//JavaScript

var maxNumber = 100;

for (var count = 1; count <= maxNumber; count++)
{
    if (count % 15 == 0) {
        console.log("fizzbuzz");
    }
    else if (count % 3 == 0) {
        console.log("fizz");
    }
    else if (count % 5 == 0) {
        console.log("buzz");
    }
    else {
        console.log(count);
    }
}

File: FizzBuzz/FizzBuzz.rb
#FizzBuzz Kata  
#if i is a multiple of 3 print "fizz"
#else if i is a multiple of 5 print "buzz"
#else if i is a multiple of 3 and a multiple of 5 print "fizzbuzz"
#else print i

def fizzbuzz(maxNumber)

	for count in 1..maxNumber
		if count % 15 == 0 then
			print "fizzbuzz"
		elsif count % 3 == 0 then
			print "fizz"
		elsif count % 5 == 0 then
			print "buzz"
		else
			print count
		end
		print "\n"
	end
end


puts fizzbuzz(1000)
